cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/sh
 set -ex
 
+CRI=${CRI:-docker}
```

Comment:
can you please do an auto detect as we have on kubevirt ?
```
function detect_cri() {
    if podman ps >/dev/null 2>&1; then echo podman; elif docker ps >/dev/null 2>&1; then echo docker; fi
}

export CRI_BIN=${CRI_BIN:-$(detect_cri)}
```
same for the rest of the files here
and then no need to export it, unless someone wants to forcefully override

better to name it please CRI_BIN (this is the name we are using on kubevirtci (case insensitive) / kubevirt)

Reply:
Done